packages/hooks/src/useSendOTPVerificationCode.ts

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { useMutation } from '@deriv/api';
+import { TSocketError } from '@deriv/api/types';
+import { localize } from '@deriv/translations';
+import useRequestPhoneNumberOTP from './useRequestPhoneNumberOTP';
+
+/** A hook for verifying Phone Number OTP and Email OTP */
+const useSendOTPVerificationCode = () => {
+    const [phone_otp_error_message, setPhoneOtpErrorMessage] = React.useState('');
+    const {
+        data,
+        mutate,
+        error: phone_otp_error,
+        isSuccess: is_phone_number_verified,
```

Comment:
isSuccess indicates whether the mutation call was successful. isnt? 
why do we map "isSuccess" to is_phone_number_verified?

Reply:
Yeappp the mutation is for sending otp value as a payload towards 'phone_number_challenge' endpoint, once it's success i map it to `is_phone_number_verified`. Should we rename it to something better? like phone_number_verified?